mysite/settings.py

```diff
@@ -57,8 +58,13 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.getenv('KPI_DB'),
```

Comment:
This is nice but also it does not belong to this PR. You should submit unrelated changes separately.